thirdparty.cmake

```diff
@@ -50,6 +50,10 @@ set(ENVOY_LIGHTSTEP_TRACER_INCLUDE_DIR "" CACHE FILEPATH "location of lighstep t
 # Extra linker flags required to properly link envoy with all of the above libraries.
 set(ENVOY_EXE_EXTRA_LINKER_FLAGS "" CACHE STRING "envoy extra linker flags")
 
+# https://github.com/miloyip/rapidjson
```

Comment:
nit: can you group code dependency in one place? (just place this before extra linker flags)